src/routes/transactions/entities/transaction-history.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ConflictType } from './conflict-type.entity';
+import { Transaction } from './transaction.entity';
+
+export class TransactionHistory {
```

Comment:
I'd use a different name here because it doesn't need to be tied to "history".

Something like `TransactionItem`.